packages/cardhost/app/templates/components/cards/cardstack/base-card/isolated.hbs

```diff
@@ -3,9 +3,14 @@
 {{!-- TODO use a sanitized version of the @card.id in the class name after we have css coming from card documents --}}
 <div
   data-test-isolated-card={{@card.name}}
-  class="isolated-card cardstack_base-card"
+  class="isolated-card cardstack_base-card template_{{@templateId}} {{@cardId}} {{@mode}}"
 >
   {{#each @card.isolatedFields key="nonce" as |field|}}
+    {{!-- TODO: This is here temporarily until we have a better solution for rendering images --}}
+    {{#if (eq field.name "image")}}
+      <img src={{field.value}} alt="" width="100%" class="isolated--hero-image">
```

Comment:
my experience has been that for responsive layout, `<img>` is always the wrong answer. you'll need to think about `contain` vs `cover` when people start resizing the page. Definitely you'll want to use a background div here.

Reply:
There’s actually `object-fit` and `object-position` css properties which do what bg image properties can do. But I’m fine with using a bg image instead too, though.